src/components/Article.jsx

```diff
@@ -42,11 +42,15 @@ export default function Article() {
           {comments.map((comment) => {
             return (
               <li key={comment.comment_id}>
-                <p>{comment.body}</p>
+                <p className="single-article--comments--comment-body">
+                  {comment.body}
+                </p>
                 <h4>
                   <span>from</span> {comment.author}
                 </h4>
-                <button>{comment.votes}</button>
+                <p className="single-article--comments--votes">
+                  {comment.votes} Votes
```

Comment:
In this case, we can extract the comments out to another comment component and then render it on article. As it is its own section of the page, it would make sense to separate this out to another component.

Reply:
Hi Hannah, I have moved comments to its own component (Comments.jsx). I have checked the API and currently there are no articles with 0 comments to be able to check, however, i have put a ternary in there for if no comments are present, and have checked it with an empty array.